README.md

```diff
@@ -19,7 +19,7 @@ The following environment variables should be provided at build time:
 - `NEXT_PUBLIC_GRAPHQL_ENDPOINT`: the URL of the OpenHexa GraphQL API
 - `NEXT_PUBLIC_SENTRY_DSN`: the [Sentry](https://sentry.io/) DSN
 - `SENTRY_AUTH_TOKEN`: A valid Sentry authentication token
-- `SENTRY_RELEASE`: a release identifier for Sentry (such as a Git tag)
+- `SENTRY_RELEASE`: a release identifier for Sentry (such as a Git tag). It will also be displayed in the footer
```

Comment:
Let's rename that to `RELEASE` (or at least `NEXT_PUBLIC_SENTRY_RELEASE` so that it corresponds to the current implementation).